content/organization/tech/accounts/index.en.md

```diff
@@ -115,3 +115,117 @@ Check out the section about [account security](/organization/tech/security/)
   one repo for your chapter, e.g., `meetup-presentations_london`. If you have code for a
   workshop you can add it to the repository or ask us to create a new repository for that. Please name 
   it location_*, e.g., `london_20161130_h2o-and-r`.
+  
+## Posit Cloud
+
+### About Posit Cloud
+
+[Posit Cloud](https://posit.cloud/) is an online platform from Posit that allows users to work on data science projects without requiring software installation
+on personal computers. This is a particularly attractive option for workshops
+as creators of a Posit Cloud space can ensure all workshop attendees
+are using the same version of both R and R packages, as well as create a directory
+with files to work from.
+
+### Posit Cloud Considerations
+
+Posit Cloud could be a wonderful addition to your R-Ladies meet-ups! Posit Cloud
+would be best suited for workshop style meet-ups where participants are encouraged
+to actively submit code or do exercises.
+
+Use of Posit Cloud requires both administrators of the Posit Cloud space and
+workshop participants to have a Posit Cloud account. This may require
+a bit of extra time to get started as users may need to create an account.
+
+There are limits to the computing capacity of the Posit Cloud account. We haven't
+tested the boundaries, but be aware that computationally heavy workshops could 
+possibly have slow performance.
+
+### Posit Cloud Use
+
+This section is for R-Ladies event organizers requesting access to Posit Cloud.
+
+All correspondence regarding Posit Cloud should be on the `posit-cloud` channel
+in the R-Ladies organizers slack, which includes both requests to access or
+support related questions.
+
+1. Request access to Posit Cloud for your event by submitting the airtable form with 
```

Comment:
What do you think of making explicit how many days in advance people should request access?